src/widgetastic_patternfly/__init__.py

```diff
@@ -393,6 +393,68 @@ def __repr__(self):
         return '{}({!r})'.format(type(self).__name__, self.locator)
 
 
+class BootstrapNav(Widget):
+    """Encapsulate a Bootstrap nav component"""
+    CURRENTLY_SELECTED = './/li[contains(@class, "active")]/a'
+    TEXT_MATCHING = './/li/a[text()={txt}]'
+    ATTR_MATCHING = './/li/a[@{attr}={txt}]'
+    TEXT_DISABLED = './/li[contains(@class, "disabled")]/a[text()={txt}]'
```

Comment:
There's nothing else inside the `a` element, so `normalize-space(...)` doesn't achieve anything more than `text()`.